src/Barcode.php

```diff
@@ -38,8 +40,14 @@ class Barcode extends AbstractValidator
     protected $messageVariables = [
         'length' => ['options' => 'length'],
     ];
-
-    /** @var array<string, mixed> */
+    /**
+     * @var array{
+     *     adapter: null|AdapterInterface,
+     *     options: null|array<string, mixed>,
+     *     length: null|int|array,
+     *     useChecksum: null|bool,
+     * }
```

Comment:
all of those are optional and not nullable, I believe.
```suggestion
     * @var array{
     *     adapter?: AdapterInterface,
     *     options?: array<string, mixed>,
     *     length?: int|array,
     *     useChecksum?: bool,
     * }
```